src/cow-react/modules/limitOrders/hooks/useRateImpact.ts

```diff
@@ -0,0 +1,15 @@
+import { useAtomValue } from 'jotai/utils'
+import { limitRateAtom } from '@cow/modules/limitOrders/state/limitRateAtom'
+import { useMemo } from 'react'
+
+export function useRateImpact(): number {
+  const { activeRate, executionRate, isLoading, isLoadingExecutionRate } = useAtomValue(limitRateAtom)
+
+  return useMemo(() => {
+    if (!activeRate || !executionRate || isLoading || isLoadingExecutionRate) return 0
+
+    const ratePercent = +activeRate.divide(executionRate).multiply(100).subtract(100).toFixed(1)
+
+    return !ratePercent || !Number.isFinite(ratePercent) || Number.isNaN(ratePercent) ? 0 : ratePercent
```

Comment:
is this correct? Number.isFinite taking a string as an argument? 

It looks like they expect a number
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite

Reply:
<img width="327" alt="image" src="https://user-images.githubusercontent.com/7122625/200784668-6df50198-e4f0-465e-b0fe-374c9eff54d8.png">
Yes, `ratePercent` is a number